lib/ansible/modules/system/cron.py

```diff
@@ -725,7 +725,7 @@ def main():
                 changed = True
 
     # no changes to env/job, but existing crontab needs a terminating newline
-    if not changed:
+    if not changed and not crontab.existing == '':
```

Comment:
It should be identical. If you want, I can change it (no too strong feelings about the semantics here).